### ISSUE TITLE
Budget bar: color unsent pledges for appointments in a different week

### ISSUE THREAD

**Issue by colinxfleming** [DCAFEngineering/dcaf_case_management#1565]
Thanks for creating an issue! Please fill out this form so we can be sure to have all the information we need, and to minimize back and forth.

* **What are we trying to do?**

From my conversation with CM Olivia after the first week of the long awaited budget bar:

> Olivia mentioned a challenge distinguishing between unsent pledges for current week vs second week -- apparently unsents count for the week they're actually in, which I think I might have mentally skipped over when writing the code. She asked that they either be colored differently to reflect being in a future week or something similar. This will bite us next week when there are unsent pledges that aren't in the rack that should count against that week -- I think marking them sent will add them in but not positive.

I think the way to do this is to add another classification of unsent pledge to the effect of 'is for future pledge period' and give it a different pledge bar color, like Olivia suggests.

* **What feature or behavior is this required for?**

proper categorization in the budget bar, so CMs don't get confused about what money is going out the door during their pledge period.

* **How could we solve this issue? (Not knowing is okay!)**

We may want to have Olivia build a few patients in sandbox so we can see what data actually looks like before we start building.

* **Anything else?**

stems from #1458

**Comment by colinxfleming** [comment]
@alisonnjones @harumhelmy @mlindberg61 @vtmeghan because the rules here are complicated (and probably DCAF-specific) as they are, I'm going to close this as a wontfix. In the event that this does need to be done and is a priority for you all, please let me know, otherwise I'm content to assume this is okay unless it comes up again organically.

**Comment by alisonnjones** [comment]
Sounds good to me. Thanks!
